### ISSUE TITLE
Possible API Issue with SABNZBD

### ISSUE THREAD

**Issue by SylentNight37** [mylar3/mylar3#1275]
After running for a while (I think about 6 hours) it seems that any comic Mylar3 tries to download via SABNZBD gets stuck trying to fetch. SABNZBD reports an API issue with a 10. IP address (mine is configured on the same host and using 127.0.0.1). It also reports an API key that doesn't match any I could find on any of my applications.

**CarePackage**

[carepackage_5f286b4fad144c786f2d89d768dd65c0990d1e76_(python3-dev).zip](https://github.com/mylar3/mylar3/files/9313126/carepackage_5f286b4fad144c786f2d89d768dd65c0990d1e76_.python3-dev.zip)

**Environment (please complete the following information):**
 - OS: Windows 10
 - Branch / commit: Python3-dev/ 3ec12af475ee1f4bbec74eec2b36fd612cc16c7f

**Additional information**
I am running the dev version of the software since I have Python 10.5 on my system. 

![SABNZBD Error](https://user-images.githubusercontent.com/11765187/184269240-979f59a6-f75d-4913-b9a4-51eda2b50cf1.png)

**Comment by barbequesauce** [comment]
10.0.0.0/8 is a private network, not publicly routable. What is the IP address on this machine?

**Comment by evilhero** [comment]
The api key that is used to send to sab is a one-time randomly generated key for each send to your client. It is not listed or given anywhere for usage outside of mylar using it behind the scenes.

As @barbequesauce mentioned the IP listed is what mylar has determined it to be based on your startup. Either you're running docker, or you're running multi-homed, so it doesn't know what to use.

You can either use the exact IP for the host IP of Mylar within the program, or you can set a variable in the config.ini to indicate which ip you are running mylar on (``return_host = http://192.168.1.1:8090`` as an example)

**Comment by barbequesauce** [comment]
Yeah, he went there... my guess is that sab is trying to connect to mylar on the public IP of the system, but mylar is configured to use localhost only (127.0.0.1). Match them up - either both communicate on localhost, or on the public IP, or on 0.0.0.0 (all interfaces)

**Comment by SylentNight37** [comment]
I'm running on a 192 network, not a 10. I have no idea where it picked up that IP form. I'm also not using multi-homing.

I'll try setting it to the explicit IP instead of using the loopback address and see if that resolves things.

That being said, it seems to work fine for a few hours, then ALL comics run into this same issue.

**Comment by SylentNight37** [comment]
I could have sworn that you'd be mistaken, but you are right on point. After trying some of your other suggestions, I just happened to run ipconfig to check out my network. Apparently there was a 10. IP address on one of the adapters. I checked the computer and saw that there was a second ethernet cable attached to the computer.

Come to find out, my daughter thought she'd have a faster connection for her games if she connected another cable. Unfortunately, we aren't running Windows server and can't setup an LACP connection.

Anyway, thank you for looking over my setup and pointing me in the right direction. Since I unplugged the cable, I haven't had any issues.
